### PR TITLE
Multiple symbolizers in one rule

### PR DESCRIPTION
**Pull request by jansule**
**Important:** Please wait with merging this PR until

- [x] [this PR of geostyler-style](https://github.com/terrestris/geostyler-style/pull/41)
was merged.

Updated parser to handle multiple symbolizers within a single rule. This follows the SLD specification, where 0..N symbolizers are allowed.

Tests and testfiles were updated accordingly.

### CODE REVIEW

File: data/slds/multi_simplelineLabel.sld [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                  <DisplacementX>0</DisplacementX>
+                  <DisplacementY>5</DisplacementY>
+                </Displacement>
+              </PointPlacement>
+            </LabelPlacement>
+            <Fill>
+              <CssParameter name="fill">#000000</CssParameter>
+            </Fill>
+          </TextSymbolizer>
+       	</Rule>
```
Comment (dnlkoch) [inline data/slds/multi_simplelineLabel.sld]: Minor: Indentation is off.

File: src/SldStyleParser.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import {
   Rule,
   ComparisonOperator,
   CombinationOperator,
-  StyleType,
+  // StyleType,
```
Comment (dnlkoch) [inline src/SldStyleParser.ts]: Remove line.

File: src/SldStyleParser.spec.ts [hunk 3]
```diff
@@ -131,13 +132,22 @@ describe('SldStyleParser implements StyleParser', () => {
...
+          expect(geoStylerStyle).toBeDefined();
+          expect(geoStylerStyle).toEqual(multi_simplelineLabel);
+        });
     });
 
+    // describe('#getStyleTypeFromSldObject', () => {
+    //   it('is defined', () => {
+    //     expect(styleParser.getStyleTypeFromSldObject).toBeDefined();
+    //   });
+    // });
```
Comment (dnlkoch) [inline src/SldStyleParser.spec.ts]: Remove commented lines.

File: src/SldStyleParser.ts [hunk 4]
```diff
@@ -94,39 +94,39 @@ class SldStyleParser implements StyleParser {
...
+  //       styleType = 'Fill';
+  //       break;
+  //     case 'LineSymbolizer':
+  //       styleType = 'Line';
+  //       break;
+  //     default:
+  //       throw new Error('StyleType could not be detected');
+  //   }
+  //   return styleType;
+  // }
```
Comment (dnlkoch) [inline src/SldStyleParser.ts]: Remove commented lines.

File: tslint.json [hunk 5]
```diff
@@ -31,19 +31,10 @@
...
-      "info",
-      "time",
-      "timeEnd",
-      "trace"
-    ],
+    "no-console": false,
     "no-consecutive-blank-lines": true,
     "no-construct": true,
-    "no-debugger": true,
+    "no-debugger": false,
```
Comment (dnlkoch) [inline tslint.json]: Development artifacts?

**Comment by jansule** [comment]
@terrestris/devs ready to merge.

**Pull request merged**